wcfsetup/install/files/lib/system/database/table/column/IEnumDatabaseTableColumn.class.php

```diff
@@ -18,15 +18,12 @@ interface IEnumDatabaseTableColumn extends IDatabaseTableColumn
      * Sets the predetermined set of valid values for the database table column and returns this
      * column.
      *
-     * @param array $values
```

Comment:
This still should qualify the type of `array`. This is likely `@param string[] $values` and likewise for the getter.